src/apps/stable/features/playback/utils/playbackSubscriber.ts

```diff
@@ -87,15 +87,15 @@ export abstract class PlaybackSubscriber {
 
         if (this.player) {
             Object.entries(this.playerEvents).forEach(([event, handler]) => {
-                if (handler) Events.off(this.player, event, handler);
+                if (handler) Events.off(this.player, event, handler.bind(this));
```

Comment:
Doesn't `bind` return a new function and `handler` won't be disconnected?

Reply:
Good call. I think this is better...